### DISCUSSION TITLE
家人们，为什么显示未登录啊

### DISCUSSION THREAD

**Discussion by cylca4869** [share121/inter-knot#5074] [委托]
网页这边github登录了也改头像和名字了，绳网这边显示未登录

**Comment by cylca4869** [comment]
![2024-09-17](https://github.com/user-attachments/assets/108c868b-d9db-4c52-ba86-b5492e8b5a3d)
难道只能发帖才看得到吗

**Comment by JT9364** [comment]
app内要重新登（我的→用户线路），登了的话要等一会或者上魔法

**Reply by cylca4869** [reply]
感谢感谢
